core-bundle/contao/themes/flexible/styles/basic.css

```
@@ -347,10 +347,31 @@ table.with-zebra tbody tr:nth-child(even) td {
 }
 
 /* Widgets */
-.widget {
-	margin-left: 15px;
-	margin-right: 15px;
-	position: relative;
+fieldset > .widget-group {
+	padding: 0 15px;
+}
+.widget-group {
+	display: grid;
+	grid-column-gap: 30px;
+	grid-template-columns: repeat(12, 1fr);
+}
+.widget-group > * {
+	grid-column: 1 / 13;
+}
+.widget-group > .m12 {
+	margin: 0;
+}
+.widget-group > .w50 {
+	grid-column: auto / span 6;
+}
+.widget-group > .w33 {
+	grid-column: auto / span 4;
+}
+.widget-group > .w25 {
+	grid-column: auto / span 3;
+}
+.widget-group > .clr {
+	grid-column-start: 1 !important;
```

```suggestion
	grid-column-start: 1;
```
We should not use `!important`. 
Instead we should replace all `grid-column: auto / span …;` with `grid-column-end: span …;`

---

Forgot that one after testing, yours is a good solution tho.

Mind that this may change over the next days but I'll prefer yours.

---

Using grid-column-end would lead to grid-column-start not being auto anymore, hence always being end + every widget being automatically in a new row.

Simply removing the `!important` is enough, however I'll move the `.clr`to the end of the css file as well.

---

@ausi 84ca874f0bc3aa1f2af4b01592ff378fac49fc76 should work tho

---

> Using grid-column-end would lead to grid-column-start not being auto anymore, hence always being end + every widget being automatically in a new row.

You just need to define `grid-column-start: auto` as well. See https://codepen.io/fritzmg/pen/WNWJOOV for example.

---

Added with your suggestions